portfolio/src/main/webapp/index.html

```diff
@@ -10,19 +10,35 @@
 
 	<body class="upper-section">
 		<div id="content">
+
       <section>
         <h1>Jenny Baik</h1>
-
         <table id="menu">
           <th onclick="Toggle('main-section')">home</th>
           <th onclick="Toggle('about-section')">about me</th>
           <th onclick="Toggle('contact-section')">contact me</th>
         </table>
+        <form action="/data" method="POST">
+          <ul>
+            <li>    
+              <label for="name">Name:</label>
+              <input type="text" id="name" name="user_name">
```

Comment:
please keep the ordering of the attributes consistent. it is type,id,name here and id,type,name below.